File: HireFlow.Backend/HireFlow.Application/Candidates/Commands/Common/CandidateDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HireFlow.Application.Candidates.Commands.Common
{
    public record CandidateDto
    (
        Guid Id,
        Guid UserId,
        string ResumeUrl,
        string PhoneNumber,
        string? LinkedInUrl,
        string? EducationLevel
    );
}

File: hireflow-client/src/app/domain/dtos/candidate.dto.ts
export interface CandidateDto {
    id : string,
    userId : string,
    resumeUrl : string,
    phoneNumber : string,
    linkedinUrl? : string,
    educationlevel? : string

}

export interface CreateProfileCommand{
    resumeUrl : string,
    phoneNumber : string,
    linkedinUrl? : string,
    educationlevel? : string   
}

export enum EducationLevel {
    Bac = 'Bac',
    BacPlus2 = 'BacPlus2',
    BacPlus3 = 'BacPlus3',
    BacPlus4 = 'BacPlus4',
    BacPlus5 = 'BacPlus5',
    PhD = 'PhD'
}   